Case Study: Great Expectations (#9562)
PostHog/posthog.com: contents/customers/great-expectations.md

---
title: 'How Great Expectations uses PostHog to improve content strategy'
customer: Great Expectations
logo: >-
  https://res.cloudinary.com/dmukukwp6/image/upload/gx_logo_light_ce286f1955.png
logoDark: >-
  https://res.cloudinary.com/dmukukwp6/image/upload/gx_logo_dark_5a1dba99f7.png
featuredImage: >-
  https://res.cloudinary.com/dmukukwp6/image/upload/posthog_great_expectations_f90a29aa4d.jpg
featuredCustomer: false
industries:
  - 'SaaS, Data'
users:
  - Growth
  - Engineering
  - Product
  - Marketing
toolsUsed:
  - PostHog Cloud
  - Insights
date: 2024-10-05
---

> Founded in 2017, [Great Expectations (GX)](https://greatexpectations.io/) helps data teams have confidence in their data. Its SaaS platform, GX Cloud, is a fully managed end-to-end solution for managing the data quality process. As you’d expect the team takes its own data seriously and decided in 2023 that they needed to gather data from across the entire user journey.

Reviewer: ```suggestion
Founded in 2017, [Great Expectations (GX)](https://greatexpectations.io/) helps data teams have confidence in their data. Its SaaS platform, GX Cloud, is a fully managed end-to-end solution for managing the data quality process. As you’d expect, the team takes its own data seriously and decided in 2023 that they needed to gather data from across the entire user journey.
```

---

> title: 'How Great Expectations uses PostHog to improve it’s content strategy'

Author: ```suggestion
title: 'How Great Expectations uses PostHog to improve content strategy'
```

customer: Great Expectations
logo: >-
  https://res.cloudinary.com/dmukukwp6/image/upload/gx_logo_light_ce286f1955.png
logoDark: >-
  https://res.cloudinary.com/dmukukwp6/image/upload/gx_logo_dark_5a1dba99f7.png
featuredImage: >-

>   https://res.cloudinary.com/dmukukwp6/image/upload/adauris_62af5ac994.png

Author: ```suggestion
  https://res.cloudinary.com/dmukukwp6/image/upload/posthog_great_expectations_f90a29aa4d.jpg
```

featuredCustomer: false
industries:
  - 'SaaS, Data'
users:
  - Growth
  - Engineering
  - Product
  - Marketing
toolsUsed:
  - PostHog Cloud
  - Insights
date: 2024-10-05
---

> Founded in 2017 and backed by [Index Ventures](https://www.indexventures.com/) and [CRV](https://www.crv.com/), Great Expectations creates software to help other teams ensure their data quality and avoid reporting anomalies. As you’d expect the team takes its own data seriously and decided in 2023 they needed the ability to gather data from across the entire user journey. 

Reviewer: ```suggestion
Founded in 2017 and backed by [Index Ventures](https://www.indexventures.com/) and [CRV](https://www.crv.com/), Great Expectations creates software to help other teams ensure data quality and avoid reporting anomalies. As you’d expect, the team takes its own data seriously and decided in 2023 they needed the ability to gather data from across the entire user journey. 
```

> “We have our marketing site, our docs website, and our product,” explains Marketing Project Manager Erica Howard. “We needed to look holistically across all three of these. Prior to PostHog we hadn’t been able to follow these full user journeys or track data in a granular way.”

Reviewer: ```suggestion
“We have our marketing site, our docs website, and our product,” explains Marketing Project Manager Erica Howard. “We needed to look holistically across all three of these. Prior to PostHog, we hadn’t been able to follow these full user journeys or track data in a granular way.”
```

> [Implementing PostHog across all three domains](/tutorials/cross-domain-tracking) also meant that this wasn’t just solving a need for the marketing team either. Soon, every team in the company was using PostHog in some capacity to gather the data they needed. 

Reviewer: ```suggestion
[Implementing PostHog across all three domains](/tutorials/cross-domain-tracking) also meant that this wasn’t just solving a need for the marketing team, either. Soon, every team in the company was using PostHog in some capacity to gather the data they needed. 
```

“Our growth team also uses PostHog, plus some of our software engineers,” says Erica. “Personally, I came in cold and didn’t know the product at all — but once I’d seen one insight setup it was fairly easy to replicate and go from there. There’s a whole lot more I _could_ do in PostHog, but getting started was pretty easy.”

### Analytics, annotations, and activation 

> As soon as tracking was implemented across the three touch-points, Great Expectations’ marketing and growth teams started creating reporting dashboards to track their most important metrics. 

Reviewer: ```suggestion
As soon as tracking was implemented across the three touchpoints, Great Expectations’ marketing and growth teams started creating reporting dashboards to track their most important metrics. 
```
